mcpserver/tools.go

```diff
@@ -108,22 +137,19 @@ func wrapToolWithClient(tool *Tool, dag *dagger.Client) *Tool {
 }
 
 func init() {
```

Comment:
🎨 **Style (Low):** Inconsistent spacing between registerTool calls - should be uniform.

```suggestion
registerTool(EnvironmentOpenTool)
	registerTool(EnvironmentCreateTool)
	registerTool(EnvironmentUpdateTool)
	registerTool(EnvironmentRunCmdTool)
	registerTool(EnvironmentFileReadTool)
	registerTool(EnvironmentFileListTool)
	registerTool(EnvironmentFileWriteTool)
	registerTool(EnvironmentFileDeleteTool)
	registerTool(EnvironmentAddServiceTool)
	registerTool(EnvironmentCheckpointTool)
```